facia/app/views/fragments/emailFrontBody.scala.html

```diff
@@ -130,11 +130,9 @@
 }
 
 @freeTextWithImage(card: ContentCard, collectionName: String) = {
-    @faciaCard(classesForCard(card)) {
-@*        here we override the alt text as 'headline' (normal alt text) may contain HTML*@
+        @*        here we override the alt text as 'headline' (normal alt text) may contain HTML*@
         @row(Seq("no-pad")){@imgFromCard(card,altTextOverride = Some(collectionName))}
```

Comment:
Just checking these shouldn't be `fullRow` like `freeText`?

Reply:
No, despite the name, `fullRow` would out a table around the image and another table around the text. This isn't necessary.

Reply:
| OK  | Cool |
| ------------- | ------------- |
| Beans  | Simon  |